utils/documentation_tests.txt

```diff
@@ -25,6 +25,7 @@ src/transformers/models/bert/modeling_tf_bert.py
 src/transformers/models/bigbird_pegasus/modeling_bigbird_pegasus.py
 src/transformers/models/big_bird/modeling_big_bird.py
 src/transformers/models/blenderbot/modeling_blenderbot.py
+src/transformers/models/blenderbot/configuration_blenderbot.py
```

Comment:
Let's move this line before `src/transformers/models/blenderbot/modeling_blenderbot.py` 🙏